easybuild/easyconfigs/c/CDO/CDO-1.8.1-CrayGNU-2016.11.eb

```diff
@@ -10,8 +10,7 @@ description = """CDO is a collection of command line Operators to manipulate and
 toolchain = {'name': 'CrayGNU', 'version': '2016.11'}
 toolchainopts = {'opt': True, 'pic': True}
 
-sources = ['cdo-%(version)s.tar.gz']
-source_urls = ['https://code.zmaw.de/attachments/download/14271/']
+sources = ['/apps/common/UES/easybuild/sources/%(nameletterlower)s/%(name)s/' + SOURCELOWER_TAR_GZ]
```

Comment:
Do you want to upgrade to :
* https://code.mpimet.mpg.de/attachments/download/15187/cdo-1.9.0.tar.gz
or even:
* https://code.mpimet.mpg.de/attachments/download/14686/cdo-1.8.2.tar.gz

Reply:
We should do that in `17.08`, it's not necessary for `2016.11`.